### ISSUE TITLE
Bug: Server API version too low

### ISSUE THREAD

**Issue by qdm12** [qdm12/deunhealth#10]
> I'm on Docker version 18.09.1, build 4c52b90

```
2021/10/17 10:31:15 ERROR Error response from daemon: client version 1.41 is too new. Maximum supported API version is 1.39
```

**Comment by qdm12** [comment]
I pushed image `qmcgaw/deunhealth:negotiate-api`, can you please try it?

If it still fails, try setting the env variable `DOCKER_API_VERSION=<your-server-version` to see if it works?

**Comment by qdm12** [comment]
Fixed by d19c4222c2627e909c1c504f5cabf893412c0cd7
